css/homepage.scss

```diff
@@ -50,6 +50,20 @@
     }
   }
 }
+#diff-data-api {
+  h2 {
+      max-width: 60em;
+      margin: 1em auto;
+      font-size: 150%;
+  }
+  .item {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    text-align: justify;
+  }
```

Comment:
No it is not redundant with semantic UI. The `grid` layout use a flex container on the `grid` element. We apply it on the cell.
With:
![screen-diff-api-data](https://cloud.githubusercontent.com/assets/5861569/16579572/a1cff316-42a0-11e6-818d-e55ab39325a5.png)

Without:
![diff-data-api-2](https://cloud.githubusercontent.com/assets/5861569/16579576/a5ee841c-42a0-11e6-8397-a132113f36d2.png)

The responsive layout is not broken